lib/main.js

```diff
@@ -69,6 +69,15 @@ function post(options) {
     });
 }
 
+function get(uri) {
+    return new Promise((resolve, reject) => {
+        request(uri, (error, response, body) => {
```

Comment:
we really should use some http lib with built-in promise support

Reply:
yea I agree. I'm kinda in the market for a good replacement for request in general. It feels bloated and no promise support. Any thoughts?

Reply:
Either node-fetch or make-fetch-happen would be my suggestions